src/components/puck/HeadingText.tsx

```diff
@@ -8,6 +8,7 @@ import {
   YextEntityField,
   YextEntityFieldSelector,
 } from "../editor/YextEntityFieldSelector.js";
+import { NumberFieldWithDefaultOption } from "../editor/NumberOrDefaultField.js";
```

Comment:
This has .js import but others you changed to .tsx. Which do we prefer? Does .tsx work?

Reply:
Ah good catch. The VSCode auto import uses .js but we do .tsx everywhere else